edx_toggles/tests/test_waffle.py

```diff
@@ -29,6 +29,17 @@ def test_constructor(self):
         self.assertEqual("module1", waffle.module_name)
         self.assertEqual(1, len(NaiveWaffle.get_instances()))
 
+    def test_toggle_methods(self):
+        waffle = NaiveWaffle("namespaced.name", "module1")
+        # test is_enabled method
+        self.assertEqual(True, waffle.is_enabled())
+        #test is_disabled method
+        self.assertEqual(False, waffle.is_disabled())
+        #test is_toggle_on method
+        self.assertEqual(waffle.is_enabled(), waffle.is_toggle_on())
```

Comment:
Nit: I think it would be quicker to read if you just stuck with True/False for the final two assertions as well.

Reply:
@robrap I was a bit confused about the third statement on updating the changelog. Just to be sure now, I only need to change from [5.1.0] to [5.2.0] right ?

Reply:
1. You'll want to add a new section to the top of the changelog for version 5.2.0. It should be a similar format as the sections below, like this example for 5.1.0, but you'll update it to read "5.2.0", and update the date, and it will have one bullet like:
```
* Added is_disabled, is_toggle_on, and is_toggle_off methods to BaseToggle.
```
 
2. [request] As I was writing the above, it made me think that we should probably rename `is_toggle_on/off` to `is_toggled_on/off` to be more consistent with the past tense of `enabled/disabled`? What do you think?

3. You'll need to change the actual version to 5.2.0 in [__init__.py](https://github.com/openedx/edx-toggles/blob/461dd555036d6d538557da106cb108749b999e09/edx_toggles/__init__.py#L5) as well. It looks like someone updated it to 5.1.1, but that doesn't affect this.

Reply:
Thanks for these updates. Marking resolved.

Reply:
Actually, unmarking as resolved, because failing tests indicate that the `is_toggle_on` rename is not complete. Sorry about that.